src/AlignmentGuides.js

```diff
@@ -251,7 +317,7 @@ class AlignmentGuides extends Component {
 				node: e.target,
 				metadata: this.state.boxes[e.target.id].metadata
 			};
-			if (e.shiftKey || (e.type === 'contextmenu' && this.state.activeBoxes.length > 1)) {
+			if (e.shiftKey || (e.type === 'contextmenu' && this.state.activeBoxes.length > 1) || ( e.target.id.indexOf('box-ms-') >= 0 && this.props?.groups?.length > 0)) {
```

Comment:
Can we add 'box-ms-' as a constant as this string is being used in multiple places?